src/main/java/com/g4s8/ghman/user/UserDetails.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright (c) 2019 Kirill Ch. (g4s8.publci@gmail.com)
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to read
+ * the Software only. Permissions is hereby NOT GRANTED to use, copy, modify,
+ * merge, publish, distribute, sublicense, and/or sell copies of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.g4s8.ghman.user;
+
+import javax.json.Json;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import org.telegram.telegrambots.api.objects.Chat;
+
+/**
+ * Details about user.
+ * @since 1.0
+ */
+public interface UserDetails {
+
+    /**
+     * User details.
+     * @return User info in json format
+     */
+    JsonObject details();
```

Comment:
@olenagerasimova can you add a `@todo` here to continue the work and expose the user details instead of a `JsonObject`? This also mean that `FromChat` won't need to create a `JsonObject` object when it is done.

Reply:
@victornoel please, check my previous answer - we have table with json field that contains telegram user data, so as long as we have such table and field we will need an object that can transform data `Chat` into `JsonObject`